### PR TITLE
Update alpha_G and alpha_I defaults

### PR DESCRIPTION
**Pull request by ARFajardo**
- [x] `make format` and `make documentation` has been run. (You may also want to run `make test`.)

This PR updates the default values for alpha_G and alpha_I.

cc: @rickecon @SeaCelo @jdebacker

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.6] - 2024-10-24 11:00:00
```
Comment (rickecon) [inline CHANGELOG.md:8]: @ARFajardo. We need to add a line at the end of this `CHANGELOG.md` file.

**Comment by rickecon** [comment]
@ARFajardo. I was able to compile the Jupyter book documentation on my machine. So you just need to add the closing dollar sign in `exogenous_parameters.md` and add the line at the end of `CHANGELOG.md`.

**Comment by ARFajardo** [comment]
Using a windows machine, it returns an error after solving the baseline steady state and does not proceed running the iterations for the transition path. 
Screenshot (252)

**Comment by rickecon** [comment]
@ARFajardo. It looks like the run script having an error was caused by your `ogphl-dev` conda environment not being updated. Your script is now running. And it runs on my machine as well. This PR is ready to be merged. Thanks to you and Frabert for this contribution.

**Pull request merged**